core/domain/question_domain.py

```diff
@@ -0,0 +1,145 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects relating to question."""
+
+from constants import constants
+from core.platform import models
+
+import feconf
+import utils
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+
+class Question(object):
+    """Domain object for a question.
+
+    Attributes:
+        question_id: str. The unique id of the question.
+        title: str. The title of the question.
+        question_data: dict. The data for the question data.
+        data_schema_version: int. The schema version for the data.
+        collection_id: str. ID of the collection.
```

Comment:
The ID of the collection containing the question.

Also, use ID (capitals) throughout the PR. Sometimes I see "id".

Reply:
Updated.